uisdk/src/test/java/com/karhoo/uisdk/util/CurrencyUtilsTest.kt

```diff
@@ -18,10 +18,58 @@ class CurrencyUtilsTest {
     fun `high price formatted correctly for individual price`() {
         val priceString = CurrencyUtils.intToPrice(
                 currency = Currency.getInstance("GBP"),
-                price = 1000)
+                price = 1000,
+                locale = Locale.UK
+        )
         assertEquals("£10.00", priceString)
     }
 
+    @Test
+    fun `when formatting Jordanian Dinnar prices, should display 3 fraction digits`() {
+        val formattedString = CurrencyUtils.intToPrice(currency = Currency.getInstance("JOD"), price = 10000, locale = Locale.UK)
+        assertEquals("JOD10.000", formattedString)
```

Comment:
I think the CircleCI Docker image might not have the requested Locale and it falls back to the one which is available or default. 
Can we try to use a more generic one e.g. `Locale.English` and see if one of the failed tests actually pass? 

You can SSH to the circleCI machine and make the changes locally on that machine if you want to speed up your development.

Reply:
I ended up refactoring the unit tests to make sure they pass everywhere.